profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -205,6 +205,15 @@ function _tmgmt_poetry_form_load_translator_settings(&$form_state, &$form, $job)
     $settings = [];
   }
 
+  // NEPT-62: Overwriting the 'delai' field properties.
+  if (isset($settings['delai'])) {
```

Comment:
I don't really understand why we overwrite the field properties when we are actually defining the form ourselves. Shouldn't this change be generic for any request?